src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,86 @@
+# Manipulating Windows
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into Management mode by positioning your cursor over the window, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+![Adjustment Mode Manipulation](/images/tiling-stacking-windows/adjustment-mode-manipulation.gif)
+
+All keyboard shortcuts available in adjustment mode can be viewed and modified in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Move, resize, and swap windows.
+
+## Auto Tiling Windows
+
+Pop!\_OS optimizes workflows using a smart window management system called Auto Tiling. Auto Tiling automatically positions and sizes windows.
+
+![Auto Tiling](/images/tiling-stacking-windows/auto-tiling.png)
+
+Enable Auto Tiling by clicking on the tile icon in the upper right corner and toggling the button. Tiling settings include options for window titles, active window hint, active window color, and gaps.
+
+![Enable Autotiling](/images/tiling-stacking-windows/enable-autotiling.png)
+
+| Option                 | Function |
+|----------------------|--------|
+| Tile Windows| Toggles the Auto Tiling feature on or off  |
+| Floating Window Exception  | Allows adding exceptions to the Auto Tiling feature allowing free positioning of specified windows.|
+| Toggle Tiling          | Enable/Disable the Auto Tiling feature with a keyboard shortcut. |
```

Comment:
I don't see why `Disable` needs to be capitalized here, it's not the first word in the sentence.

You might consider saying "Enable or disable" instead of using a slash to better match the first row in this table.

Also, the rest of these descriptions use present-tense and not imperative, so you might say "Enables or disables" with an `s` at the end of each.